scripts/goreleaser

```diff
@@ -22,6 +24,17 @@ docker build -f ./build/Dockerfile.goreleaser -t ecctl:goreleaser .
 docker run --rm -ti --privileged \
     -e OWNER="${OWNER}" -e REPO="${REPO}" -e GO111MODULE=on \
     -e GITHUB_TOKEN="${GITHUB_TOKEN}" -e VERSION="${VERSION}" \
-    -v "$GOPATH:/go" \
+    -e BUILT="${BUILT}" -v "$GOPATH:/go" \
     -v "$(pwd):/go/src/github.com/${OWNER}/${REPO}" \
     -w "/go/src/github.com/${OWNER}/${REPO}" ecctl:goreleaser ${*}
+
+# Upload the binaries and the checksums.
+for f in dist/*.{tar.gz,deb,rpm,txt}; do
+    aws --profile ecsecurity s3 cp --acl bucket-owner-full-control \
```

Comment:
not sure if we want to expose these. I mean these don't mean anything to people outside `elastic`. WDYT?

Reply:
We (the maintainers) are the only ones who are going to be releasing this and it's our profile, which is only a name here, not sure there's any harm

Reply:
Completely harmless - just wondering if we want to follow this path 😄

Reply:
ah - of course, from my side I was curious as to if I had missed something